drivers/rilmodem/gprs.c

```diff
@@ -128,7 +134,12 @@ void ril_gprs_set_ia_apn(struct ofono_gprs *gprs, const char *apn,
 	struct cb_data *cbd;
 	struct parcel rilp;
 
-	if (!ofono_modem_get_boolean(gd->modem, MODEM_PROP_LTE_CAPABLE)) {
+	/*
+	 * FIXME for the moment we use ril version to avoid the need to set
+	 * the env var OFONO_RIL_RAT_LTE in latest modems.
+	 */
```

Comment:
Perhaps it might be worth a comment stating the RIL v11 first appeared in Marshmallow?